### ISSUE TITLE
Missing numpy include

### ISSUE THREAD

**Issue by ckoerber** [gplepage/vegas#7]
Hello @gplepage ,

I have encountered the following issue when pip installing `vegas` for python2.7 and python3.6: When compiling `src/vegas/_vegas.c` I receive the error message 
```
src/vegas/_vegas.c:538:10: fatal error: 'numpy/arrayobject.h' file not found 
#include "numpy/arrayobject.h"
```
Numpy (1.13.3) and Cython (0.27.3) are installed. The compiler includes `-I/usr/local/include` but not my `numpy.get_include()`. Fixing `ext_modules = cythonize('src/vegas/_vegas.pyx')` to 
```
ext_modules = cythonize('src/vegas/_vegas.pyx')
ext_modules[0].include_dirs += include_dirs
``` 
resolved my issues.

Thank you for providing the `vegas`package and best regards,

Christopher Körber

**Comment by gplepage** [comment]
Thanks for the report. 

cython's documentation suggests modifying ``ext_modules = cythonize('src/vegas/_vegas.pyx')`` to

``ext_modules = cythonize('src/vegas/_vegas.pyx', include_path=include_dirs)``

Do you know if that fixes your problem? 

I can't cause your problem on my machine, even with the same ``numpy`` and ``cython``, so I can't test whether this fixes things on your machine. 

Peter

**Comment by ckoerber** [comment]
Unfortunately not. Apparently `cythonize`s `include_path` keyword does not change the `include_dirs` of the returned `setuptools.extension`:
```
include_dirs = [numpy.get_include()]
print(include_dirs)
ext_modules = cythonize('src/vegas/_vegas.pyx', include_path=include_dirs)
print(ext_modules)
print(ext_modules[0].include_dirs)
```
returns 
```
['/usr/local/lib/python3.6/site-packages/numpy/core/include']
[<setuptools.extension.Extension('vegas._vegas') at 0x10e294240>]
[]
```
It seems related to https://github.com/cython/cython/issues/1480.

**Comment by gplepage** [comment]
Very helpful. Thanks.

I have redone setup.py to explicitly hand ``include_dirs`` to the extension, as is done for the gvar module (which I am assuming you built without problem). The corrected vegas is version 3.3.2.

**Comment by ckoerber** [comment]
Yes `gvar` installed without issues on my system and now `vegas` does as well. Thank you very much for your quick and efficient support.
